test_codecept/e2e/features/pageObjects/searchPage.js

```diff
@@ -159,5 +159,37 @@ class SearchPage {
   async waitForSearchWithNoResults(){
     await BrowserWaits.waitForElement(this.noResultsNotification); 
   }
+
```

Comment:
to avoid confusion, can we move NoC specific functions  to its own page object file